static/src/stylesheets/module/facia/_item.scss

```diff
@@ -308,6 +308,54 @@ $fc-item-gutter: $gs-gutter / 4;
     display: none;
 }
 
+.fc-item__slideshow {
+    img {
+        @include mq($until: tablet) {
+            &:nth-child(1n+2) {
+                display: none;
```

Comment:
seems not - in chrome at least it only seems to download when it needs them - however that does cause some tearing as it fades in